packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -52,6 +53,14 @@ const Accordion = ({
   animationTiming,
   ...props
 }: AccordionProps) => {
+  const elementId = useId();
+  const reducedMotionQuery = '(prefers-reduced-motion: reduce)';
+  const reducedMotionEnabled = useMediaQuery(reducedMotionQuery, true);
```

Comment:
We're just using `matchMedia` inside of a `useLayoutEffect` elsewhere, see: https://github.com/dequelabs/cauldron/blob/develop/packages/react/src/components/TwoColumnPanel/TwoColumnPanel.tsx#L181

If we want to consolidate, that's fine but we should try to be consistent.